### ISSUE TITLE
Level-up message (in the Infobox) has an incorrect FP amount

### ISSUE THREAD

**Issue by scibuff** [dsiekiera/foe-helfer-extension#771]
The calculation for returned/gained forge points is incorrect. The message says 0 FPs have been returned.

![image](https://user-images.githubusercontent.com/722049/78842907-9addbb00-7a01-11ea-88ad-c0b9bb5283a2.png)

console error:
```
_main.js?v=2.4.4:379 TypeError: Cannot read property 'find' of null
    at Object.NoticeIndicatorService_getPlayerNoticeIndicators (infoboard.js?v=2.4.4:347)
    at Object.BoxContent (infoboard.js?v=2.4.4:185)
    at infoboard.js?v=2.4.4:58
    at WebSocket.wsMessageHandler (_main.js?v=2.4.4:377)
wsMessageHandler @ _main.js?v=2.4.4:379
```

This is related to #762

Window 10
Chrome/80.0.3987.163
en9 (Jaims)
2.4.4 - English

**Comment by syjytg** [comment]
How do you get the info box to fill? Mine is always empty.

**Comment by scibuff** [comment]
> How do you get the info box to fill? Mine is always empty.

The infobox is always empty when you open it. You need to wait until events in the game happen ...

**Comment by scibuff** [comment]
This is now fixed on Beta (#1107) and will be in the next release
